src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPageFeed/ActionDetailsPageFeed.tsx

```diff
@@ -17,14 +17,17 @@ const ActionDetailsPageFeed = ({
   actionData,
 }: // networkEvents,
 ActionsPageFeedProps) => {
-  const events = ACTIONS_EVENTS[actionData.type];
+  const events =
+    JSON.parse(actionData.individualEvents as string) ||
+    ACTIONS_EVENTS[actionData.type];
```

Comment:
I think this should be refactored so all action events are of the same type (instead of having some as just strings and others like this as objects). 

At the very least we want:

``` graphql

type ActionEvent {
 name: String!
}
```
And then we can add all the rest as optional.

That would mean you could pass the event down into ActionsPageEvent directly, so you wouldn't then have to "find" it again inside `getColonyRoleSetTitleValues`

Reply:
Maybe not in this issue but it'd be nice to make this a bit more coherent